src/goto-programs/remove_exceptions.h

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************\
+
+Module: Remove function exceptional returns
+
+Author: Cristina David
+
+Date:   December 2016
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_REMOVE_EXCEPTIONS_H
+#define CPROVER_GOTO_PROGRAMS_REMOVE_EXCEPTIONS_H
+
+#include <goto-programs/goto_model.h>
+
+#define EXC_SUFFIX "#exception_value"
```

Comment:
This might be needed outside remove_exceptions.cpp. For instance, I use it in java_entry_point.cpp in PR #578.